change/@ni-nimble-components-48e9cade-64e1-47a5-a616-c2e2a930a2b1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> The referenced bug doesn't really answer the question of where sort/grouping indicators (and future menu dropdown) should go when right-aligning the header. For now, I have assumed we should shift them to the left side of the column label.

Looks like PO and Design have thought about... so guess should make sure the changes align with their expectations
![image](https://github.com/ni/nimble/assets/1588923/91f610ae-4cac-4d01-ad1d-d3448bfd7f65)

Think it's strange to be in Owner's without those questions answered (I don't own the answers) so I'll mark the PR reject 🤷‍♀️

Reply:
Brandon and I discussed - the sorting and grouping indicators should be mirrored when the alignment shifts

![image](https://github.com/ni/nimble/assets/1458528/3d0943aa-9284-4aeb-874e-978ab765cd0f)